File: algorithms/Convert Integer to the Sum of Two No-Zero Integers/Convert Integer to the Sum of Two No-Zero Integers.js
var getNoZeroIntegers = function(n) {
   for(let i=1;i<=n;i++){
       if(!haveZero(i) && !haveZero(n-i)){
           return [i,n-i]
       }
   }
};

const haveZero = (n) =>{
    let copy = n;
    while(copy>0){
        if(copy%10===0){
            return true
        }
        copy=Math.floor(copy/10)
    }
    return false
}

File: algorithms/Convert Integer to the Sum of Two No-Zero Integers/Convert Integer to the Sum of Two No-Zero Integers.java
class Solution {
    public int[] getNoZeroIntegers(int n) {
        int B;
        for (int A = 1; A < n; ++A) {
            B = n - A;
            if (!(A + "").contains("0") && !(B + "").contains("0"))
            return new int[] {A, B};
    }
        return new int[]{};
}
}
